sycl/include/CL/sycl/half_type.hpp

```diff
@@ -146,8 +160,8 @@ class half {
 #ifndef __SYCL_DEVICE_ONLY__
   // Since StorageT and BIsRepresentationT are different on host, these two
   // helpers are required for 'vec' class
-  half(const detail::host_half_impl::half &rhs) : Data(rhs) {};
-  operator detail::host_half_impl::half() const { return Data; }
+  constexpr half(const detail::host_half_impl::half &rhs) : Data(rhs){};
```

Comment:
Code style: A space was removed between `Data(rhs)` and `{}` - I wonder why it was done, because it kind of contradicts the styl of lines 94 and 156

Reply:
I think those spaces/etc are strictly controlled by clang-format.

Reply:
Fair enough, but I still wonder why is code style different?

Reply:
I might be wrong about this, but I think we only check clang-format has been applied to the lines that change.  There may be lines (or complete files) that haven't been formatted by clang-format, and they won't be updated unless they change.

Reply:
> but I think we only check clang-format has been applied to the lines that change.

That is correct, but from what I see on "Files changed" page is that both lines 94 and 163 were changed and to me they seem to have inconsistent formatting - that's what I was wondering about